src/bld/java/rife/BldBuild.java

```diff
@@ -132,11 +132,9 @@ public void zipBld()
                             f.perms(0755);
                         });
                     });
-                    b.dir("lib", l -> {
-                        l.file("bld-wrapper.jar", f -> f.move(path(tmp, "lib", "bld", "bld-wrapper.jar")));
-                    });
+                    b.dir("lib", l -> l.file("bld-wrapper.jar", f -> f.move(path(tmp, "lib", "bld", "bld-wrapper.jar"))));
```

Comment:
I prefer this to remain on three lines since it clearly separated the operation for the location, and matches the two other operation formats above.